bucketlist_app/views.py

```diff
@@ -1,6 +1,129 @@
-from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth import login, authenticate, logout
+from django.contrib.auth.models import User
+from django.shortcuts import render, redirect
+from django.contrib.auth.decorators import login_required
+
+from .forms import LoginForm, SignUpForm, BucketListForm, ItemForm
+from .helper import get_or_none
+from .models import Bucketlist, Items
+from .pusher import pusher_client
+
 
 # Create your views here.
-def index():
+def index(request):
```

Comment:
This `function/view` can be optimized though, I can see code repetition though

Reply:
I've made it better by removing repetitions.